src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react';
+import Alert from './Alert';
+import Box from '../box';
+import Button from '../button';
+import { addStoryInGroup, MID_LEVEL_BLOCKS } from '../../../.storybook/utils';
+import { TextBody } from '../typography';
+
+export default {
+  component: Alert,
+  title: addStoryInGroup(MID_LEVEL_BLOCKS, 'Alert'),
+  parameters: {
+    docs: {
+      source: {
+        type: 'code',
+      },
+    },
+  },
+};
+
+export const DefaultStory = (args) => {
+  const [active, setActive] = useState(false);
+
+  const toggleAlert = () => {
+    setActive(!active);
+  };
+
+  return (
+    <Box>
+      <Button onClick={toggleAlert} label="Open an alert"/>
+      <Alert
+        {...args}
+        active={active}
+        onEscKeyDown={toggleAlert}
+        onOverlayClick={toggleAlert}
+      >
+        <TextBody>I am the alert body text</TextBody>
+      </Alert>
+    </Box>
+  );
+};
+
+DefaultStory.args = {
+  primaryAction: {
+    label: 'Confirm',
+    onClick: () => console.log('primaryAction.onClick'),
+  },
+  secondaryAction: {
+    label: 'Cancel',
+    onClick: () => console.log('secondaryAction.onClick'),
+  },
+  title: 'Alert title',
+};
+
+export const ConfirmationAlert = (args) => {
```

Comment:
Can't we use those "knobs" (or whatever we use now) instead of creating separate stories?

Reply:
I've discussed this with the design team: preferably we have separate stories to show the 3 different use cases. 🤷‍♂️